rundeckapp/build.gradle

```diff
@@ -543,3 +560,12 @@ if (project.hasProperty('signing.keyId') && project.hasProperty('signing.passwor
     }
 }
 
+normalization {
+    runtimeClasspath {
+        ignore '**/grails.build.info'
```

Comment:
can you tell me more about the contents here and why we had to ignore the whole file? `grails.build.info`